Calculator/operations.py

```diff
@@ -41,13 +41,10 @@ def ceiling(a):
         return int(a) + 1
 
 def power(a,b):
-    #a raised to the power of b
-    c = a
-    for x in range(1,b):
-        a = a*c
-    return a
-    # only works for int powers
-    # meed to fix for power of 0
+    if a == 0 and b == 0:
```

Comment:
Why is it important for these numbers not to be 0?

Reply:
In python 0 ** 0 == 1, I  prefer undetermind

Reply:
That's fair but in future projects be sure to think about the user that you're intending to use your tool. If you are making this solely for yourself then there is no problem. In our case it's a bit contrived because we're making one of the most ubiquitous programs in the world so people aren't tied to us. 

In the future, make sure you can remember all the cases that are potentially unintuitive to people using your program. You will have to document your intended behavior so your users don't hate you. :-p

Reply:
I honestly thought that's how calculators normally work : selffacepalm:

Reply:
When you are ready to merge the PR let me know so I can give a proper review/approval. 👍